src/CSnakes.Tests/GeneratedSignatureTests.cs

```diff
@@ -226,4 +229,17 @@ public void TestUnionReturnType(string returnType)
         var module1 = ModuleReflection.MethodsFromFunctionDefinitions(functions1).ToImmutableArray();
         Assert.Single(module1);
     }
+
+    [Theory]
+    [InlineData("def foo(  # csharp: ignore\r\n        bar,\r\n        baz,\r\n    ):\r\n    pass\r\n")]
+    [InlineData("def hello(): # csharp: ignore ...\n")]
+    [InlineData("def foo():  # type: ignore # csharp: ignore\r\n    pass\r\n\r\ndef bar ():  # csharp: ignore # type: ignore\r\n    pass")]
+    public void TestCSharpIgnoreCommentFunction(string code)
+    {
+        SourceText sourceText = SourceText.From(code);
+        Assert.True(PythonParser.TryParseFunctionDefinitions(sourceText, out var functions, out var errors));
+        Assert.Empty(errors);
+        var module = ModuleReflection.MethodsFromFunctionDefinitions(functions).ToImmutableArray();
+        Assert.Empty(module);
```

Comment:
Wouldn't this be sufficient?

```suggestion
        Assert.Empty(functions);
```